packages/components/src/lib/controllers/app-controller.js

```diff
@@ -121,7 +121,10 @@ export default class AppController {
      * @type {UIEventController}
      */
     this._evc = new UIEventController()
-    this._selc = new SelectionController(this.getDefaultLangCode.bind(this))
+    this._selc = new SelectionController({
+      getDefaultLangCodeFn: this.getDefaultLangCode.bind(this),
+      getPrioritizeDefaultLanguageFn: this.getPrioritizeDefaultLanguage.bind(this)
+    })
```

Comment:
This is the controller that operates with text selection.
These methods are closures of Vuex.Store variables.
I passed this way because SelectionController does not have a reference to the store.
And I decided to integrate it the same way it was done before